static/src/javascripts/projects/common/modules/commercial/contributions-utilities.js

```diff
@@ -212,17 +212,7 @@ const makeABTestVariant = (
                 name: parentTest.id,
                 variant: id,
             },
-        }),
-        supportCustomURL = null,
-        supportURL = addTrackingCodesToUrl({
-            base: supportCustomURL || selectBaseUrl(),
-            componentType: parentTest.componentType,
-            componentId: campaignCode,
-            campaignCode,
-            abTest: {
-                name: parentTest.id,
-                variant: id,
-            },
+            geolocation: geolocationGetSync(),
```

Comment:
I don't understand why `supportURL` can be removed - that leaves us with just the `membershipURL` (https://membership.theguardian.com/supporter) which **doesn't** redirect to support for UK people. If one of them can go, is it not `membershipURL` since support redirects to membership? And if one of them does go, I think I'd rather it was in a separate PR

Reply:
ohhh I see... because the `membershipBaseURL` gets overridden by `selectBaseUrl`, and then only the `membershipURL` is used in the templates.

Some of this code is incredibly confusing and left behind from the baseline tests from several months ago. We should definitely clean it up and consider whether we can remove those switches (`usSupporterTrafficToNewSupportFrontend`, `ukSupporterTrafficToNewSupportFrontend`).

Reply:
but maybe not in this PR

Reply:
I totally agree, I was planning on doing it after but it probably makes more sense to do it first. Here is a PR which removes the switches from the codebase https://github.com/guardian/frontend/pull/18564